app/services/action_queue.rb

```diff
@@ -146,7 +148,9 @@ def transaction_payload
           },
           order: {
             amount:       data[:amount],
-            currency:     data[:currency]
+            currency:     data[:currency],
+            bank_name:    data[:bank_name],
+            account_number_ending: data[:account_number_ending]
```

Comment:
I might well just be misunderstanding code, but if we're trying to pass bank_name and account_number_ending to the order table in AK, I don't think that's going to work as those columns don't exist. We need to pass it to custom action fields like we do for mobile data I think.